test/token/ERC20/TokenVesting.test.js

```diff
@@ -18,9 +17,10 @@ contract('TokenVesting', function ([_, owner, beneficiary]) {
   const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
 
   beforeEach(async function () {
-    this.start = (await latestTime()) + duration.minutes(1); // +1 minute so it starts after contract instantiation
-    this.cliffDuration = duration.years(1);
-    this.duration = duration.years(2);
+    // +1 minute so it starts after contract instantiation
```

Comment:
This sort of thing feels brittle. I'm not saying we change it here, but now that our eyes are on it, can we think of alternative ways to achieve the same thing?

Reply:
Also thinking about how we can make the whole time testing work on an actual node at some point... Perhaps the more robust thing would be to have a mock of a timestamp `ts` for which `ts.greaterThan(now)` is always true.

Reply:
Woulnd't the mock thing need to happen at the node, though? I'm not sure how we can get around the contract executing the `TIMESTAMP` instruction from the suite.

Reply:
One idea that's been going around is to have a `Time` base contract that all contracts inherit which can be swapped out for a mock... See #355.